### PR TITLE
[pt-br] Translated page: pod-security-policy #13939

### PR DESCRIPTION
**Pull request by edsondearaujo**
Translation of the Pod Security Policies page into Brazilian Portuguese.

Related Issue:
#13939

### CODE REVIEW

File: content/pt-br/docs/concepts/security/pod-security-policy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+<!-- visão geral -->
+
+{{% alert title="Funcionalidade removida" color="warning" %}}
+PodSecurityPolicy foi [descontinuada](/blog/2021/04/08/kubernetes-1-21-release-announcement/#podsecuritypolicy-deprecation)
+no Kubernetes v1.21, e removido do Kubernetes v1.25.
+{{% /alert %}}
+
+Em vez de usar PodSecurityPolicy, você pode aplicar restrições semelhantes em pods usando
```
Comment (MrErlison) [inline content/pt-br/docs/concepts/security/pod-security-policy.md:14]: Minor: "Pods" instead of "pods"

**Comment by MrErlison** [comment]
Hello @edsondearaujo! Thanks for your contribution!

**Review by MrErlison — APPROVED** [review]

File: content/pt-br/docs/concepts/security/pod-security-policy.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Políticas de Segurança do Pod
+content_type: conceito
+weight: 30
+---
+
+<!-- visão geral -->
+
+{{% alert title="Funcionalidade removida" color="warning" %}}
+PodSecurityPolicy foi [descontinuada](/blog/2021/04/08/kubernetes-1-21-release-announcement/#podsecuritypolicy-deprecation)
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/security/pod-security-policy.md:10]: Minor: little inconsistency between the chosen gender for PodSecurityPolicy. Here, you used "descontinuada", but in the next sentence, you used "removido". Update one of them to maintain consistency.

File: content/pt-br/docs/concepts/security/pod-security-policy.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Em vez de usar PodSecurityPolicy, você pode aplicar restrições semelhantes em Pods usando
+um ou ambos:
+
+- [Admissão de segurança do pod](/docs/concepts/security/pod-security-admission/)
+- um plug-in de admissão de terceiros, que você mesmo implanta e configura
+
+Para obter um guia de migração, consulte [Migre de PodSecurityPolicy para o controlador de admissão PodSecurity embutido](/docs/tasks/configure-pod-container/migrate-from-psp/).
+Para obter mais informações sobre a remoção desta API,
+Veja [Descontinuação de PodSecurityPolicy: passado, presente e futuro](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/).
```
Comment (stormqueen1990) [inline content/pt-br/docs/concepts/security/pod-security-policy.md:22]: Minor: "Veja" should use lowercase "v" (sentence continuation).

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @edsondearaujo!

Thank you very much for your contribution. 🎉 

I found two very minor typos, could you please take a look and fix accordingly?

Please don't hesitate to reach out to us on the `#kubernetes-docs-pt` channel on Kubernetes Slack, should you have any questions.

**Review by MrErlison — APPROVED** [review]

**Comment by MrErlison** [comment]
/cc @edsoncelio 

PTAL

File: content/pt-br/docs/concepts/security/pod-security-policy.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+title: Políticas de Segurança do Pod
+content_type: conceito
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/security/pod-security-policy.md:3]: content_type doesn't needs to be translated so conceito -> concept

**Review by edsoncelio — COMMENTED** [review]
Hi @edsondearaujo - just a small fix, after that will be lgtm

File: content/pt-br/docs/concepts/security/pod-security-policy.md (hunk 4 above)
Comment (edsondearaujo) [reply content/pt-br/docs/concepts/security/pod-security-policy.md]: done

**Pull request merged**